test/functional/apps/discover/_discover.js

```diff
@@ -114,7 +114,7 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.discover.waitUntilSearchingHasFinished();
 
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
-        expect(Math.round(newDurationHours)).to.be(24);
+        expect(Math.round(newDurationHours)).to.be(26);
```

Comment:
why this change?

Reply:
The layout changed, and so the width of the date histogram, but what didn't change is the width of the brush. Because the date histogram is a bit smaller, a longer time range is selected. that's why more hours were selected